apps/settings/js/modules/apps_cache.js

```diff
@@ -71,7 +71,18 @@ define(function(require) {
 
       mozAppsMgmt.oninstall = function(evt) {
         var installedApp = evt.application;
-        self._apps.push(installedApp);
```

Comment:
the name `installedApp` confused me, how about name it as `newApp`, since we do push or replace the applist with this newly installed app